devices.js

```diff
@@ -5977,6 +5977,42 @@ const devices = [
         extend: preset.light_onoff_brightness(),
         meta: {turnsOffAtBrightness1: true},
     },
+    {
+        zigbeeModel: ['RC 110'],
+        model: 'RC 110',
+        vendor: 'Innr',
+        description: 'Remote Control',
+        fromZigbee: [fz.ignore_basic_report,
+            fz.ignore_command_off,
+            fz.command_on_state, fz.command_off_state,
+            fz.innr_rc110_levelCtrl_move, fz.innr_rc110_levelCtrl_stop, fz.innr_rc110_levelCtrl_step,
```

Comment:
Please use the existing generic converters instead of adding custom ones, e.g. `fz.innr_rc110_levelCtrl_stop` -> `fz.command_stop`.

Reply:
i had some issues with the existing implementation. it was impossible to have an individual simulated brightness level per endpoint.
maybe the existing one can be refactored to accept options to shape the payload.

Reply:
> maybe the existing one can be refactored to accept options to shape the payload.

 yes please, this will save a lot of code.

Could you update it?

Reply:
Yes, i will look into this.

Reply:
I have finished refactorings. Please let me know about any suggestions.